index.js

```diff
@@ -14,42 +14,43 @@ class ImmediateStore {
     this.mem = []
   }
 
-  put (index, buf, cb) {
+  put (index, buf, cb = () => {}) {
     this.mem[index] = buf
     this.store.put(index, buf, err => {
       this.mem[index] = null
-      if (cb) cb(err)
+      cb(err)
     })
   }
 
-  get (index, opts, cb) {
+  get (index, opts, cb = () => {}) {
```

Comment:
Yes this is consistent with other implementations, but making `cb` optional doesn't make much sense for `get`